### PR TITLE
Fix/borrow non default decimals

### PR DESCRIPTION
**Pull request by peetzweg**
### Fixes

Various fixes for tokens with decimals !=18 and overall improvements to the borrow experience
Show overall correct values for supply, liquidity.

Screenshot 2022-09-08 at 17 15 29

### Improved UX

Screenshot 2022-09-08 at 17 12 57

Screenshot 2022-09-08 at 17 12 54

### CODE REVIEW

File: packages/ui/components/pages/Fuse/Modals/PoolModal/AmountSelect.tsx [hunk 1]
```diff
@@ -307,45 +289,84 @@ const AmountSelect = ({
...
+                    <Row
+                      width="100%"
+                      mt={4}
+                      mainAxisAlignment="flex-end"
+                      crossAxisAlignment="center"
+                    >
+                      <Alert status="info" pb={4}>
+                        <AlertIcon />
+                        {`For safety reasons, you need to borrow at least a value of $${
+                          minBorrowUSD ? minBorrowUSD?.toFixed(2) : 100
```
Comment (carlomazzaferro) [inline packages/ui/components/pages/Fuse/Modals/PoolModal/AmountSelect.tsx:329]: Should we set this in the SDK side as a config parameters? Not necessarily in this PR

File: packages/ui/components/pages/Fuse/Modals/PoolModal/MaxBorrowSlider.tsx [hunk 2]
```diff
@@ -120,7 +121,7 @@ function MaxBorrowSlider({
           )}
         </HStack>
         <Text>
-          ${(borrowLimit * Number(utils.formatUnits(underlyingPrice)) * price).toFixed(2)}
+          ${(borrowLimit * Number(utils.formatUnits(asset.underlyingPrice, 18)) * price).toFixed(2)}
```
Comment (carlomazzaferro) [inline packages/ui/components/pages/Fuse/Modals/PoolModal/MaxBorrowSlider.tsx:124]: here as well

File: packages/ui/constants/index.ts [hunk 3]
```diff
@@ -39,6 +39,7 @@ export const ADMIN_FEE = {
 
 export const POOLS_PER_PAGE = 6;
 
+// TODO: We should replace this with NATIVE_DECIMALS from the @midas-capital/chains package
```
Comment (carlomazzaferro) [inline packages/ui/constants/index.ts:42]: I suppose all EVMs are like this!

File: packages/ui/components/pages/Fuse/Modals/PoolModal/AmountSelect.tsx (hunk 1 above)
Comment (carlomazzaferro) [reply packages/ui/components/pages/Fuse/Modals/PoolModal/AmountSelect.tsx:329]: Ah, sorry! Yes, that's perfect.

File: packages/ui/components/pages/Fuse/Modals/PoolModal/MaxBorrowSlider.tsx [hunk 4]
```diff
@@ -88,7 +84,12 @@ function MaxBorrowSlider({
               width={`${borrowedPercent}%`}
             >
               <SliderMark value={borrowedPercent} mt={4} ml={-4} fontSize="sm">
-                ${(borrowedAmount * Number(utils.formatUnits(underlyingPrice)) * price).toFixed(2)}
+                $
+                {(
+                  borrowedAmount *
+                  Number(utils.formatUnits(asset.underlyingPrice, 18)) *
```
[the remark this answers is not in the document]
Comment (peetzweg) [reply packages/ui/components/pages/Fuse/Modals/PoolModal/MaxBorrowSlider.tsx:90]: Yep, underlyingPrices aka oracle prices are always 18 decimals. 
Made sure to used hardcoded 18 value to show that it's by intention and not relying on the default value which is also 18 as of now.

**Comment by carlomazzaferro** [comment]
Overall this is a sweet PR. Looking good, appreciate the thoughtfulness in the changes, refactors, and additions.

PS: what was the the issue that caused USDC to be just slightly off? Still cannot find the specific reason

File: packages/ui/constants/index.ts (hunk 3 above)
Comment (peetzweg) [reply packages/ui/constants/index.ts:42]: Good point. Not sure if EVM compatibility enforces native assets to have 18 decimals but could be!

**Pull request merged**